machine/intel-nuc

```diff
@@ -7,7 +7,7 @@ RUN apk --no-cache add \
 ##
 # Build libcec for HDMI-CEC
 ARG LIBCEC_VERSION=4.0.4
-RUN apk add --no-cache eudev-libs p8-platform \
+RUN apk add --no-cache eudev-libs libva-intel-driver p8-platform \

```

Comment:
Please attach it under usbutils:
```
        usbutils \
        libva-intel...
```

Reply:
Oops, missed that.

it's only two packages there so not a big deal, but some recommend listing packages in alphabetical order?
```
libva-intel-driver \
usbutils
```
up to you